docs/user/build-customization.rst

```diff
@@ -362,12 +362,14 @@ Where to put files
 It is your responsibility to generate HTML and other formats of your documentation using :ref:`config-file/v2:build.commands`.
 The contents of the ``_readthedocs/<format>/`` directory will be hosted as part of your documentation.
```

Comment:
```suggestion
The contents of the ``$READTHEDOCS_OUTPUT/<format>/`` directory will be hosted as part of your documentation.
```

Reply:
I was kind of wondering if leaving this exposed would be helpful somehow. In case people need to hard-code it. But let's just remove it then :+1:

Reply:
Yeah, hardcoding it is exactly what we want to avoid. We may need to change this path in the future and it will break all the projects that are hardcoding this value.